client/src/components/Auction.jsx

```diff
@@ -15,6 +15,7 @@ class Auction extends Component {
     //if logged in, grab all info and redirect to payment page.
   componentWillMount() {
     let auctionId = this.props.match.params.auctionId;
+    console.log('this.props: ', this.props);
```

Comment:
I think we should try and remove our logs before making PRs. I'm guilty of it as well. After this I think it's good to merge.